app/assets/js/scripts/lottery.js

```diff
@@ -3,6 +3,8 @@
 
 //Requires
 const { lotteryService } = require("./assets/js/lotteryapiservice");
+// const { whitelistService } = require("./assets/js/whitelistservice");
```

Comment:
moon2BOO

Reply:
I can comment in code why I left this here. We leverage this service but this is a case where the service is available on Global Scope because the wonky way this project works :eelsbadman: